cmasher/utils.py

```diff
@@ -233,6 +239,102 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    nodes: Optional[list[float]] = None,
+    n_rgb_levels: int = 256,
+) -> LinearSegmentedColormap:
+    """Create a composite matplotlib colormap by combining multiple colormaps.
+
+    Parameters:
```

Comment:
```suggestion
    Parameters
    ----------
```

Reply:
Sure I would revise those to keep consistent with other functions.